backend/src/test/java/org/eclipse/tractusx/semantics/hub/ModelsApiTest.java

```diff
@@ -165,9 +165,7 @@ public void testGenerateOpenApiEndpointSpecExpectSuccess() throws Exception {
-      ;
+            .andExpect(jsonPath( "$.info.title", containsString( "Movement" )));
```

Comment:
Compare the content?

Reply:
Hi @tunacicek  ,

Thanks for the review comments.

Please note that changes in the generated OpenApi specification were causing failures due to missing properties and version adjustments. The ESMF team recommended using assertions for specific values instead of relying on the complete JSON structure to ensure future compatibility with ESMF SDK updates.